jupyter-js-widgets/webpack.config.js

```diff
@@ -1,4 +1,5 @@
 var version = require('./package.json').version;
+require('es6-promise').polyfill();
```

Comment:
Is this the right place to put this?  Seems like you meant to put it in embed-webpack.js